lib/generic/PoleEffect.cpp

```diff
@@ -0,0 +1,55 @@
+#include "PoleEffect.hpp"
+
+#include <FastLED.h>
+
+CHSV GetColor(uint8_t n, long timeMs) {
```

Comment:
Yes, this should be static. Making it static limits its visibility to only this translation unit, i.e. file.

Reply:
Ack: Moved to unnamed namespace per what I think is @davidstone's suggestion